token-lending/cli/Cargo.toml

```diff
@@ -9,15 +9,15 @@ repository = "https://github.com/solana-labs/solana-program-library"
 version = "0.1.0"
 
 [dependencies]
-clap = "2.33.3"
-solana-clap-utils = "1.7.12"
-solana-cli-config = "1.7.12"
-solana-client = "1.7.12"
-solana-logger = "1.7.12"
-solana-sdk = "1.7.12"
-solana-program = "1.7.12"
+clap = "=2.34.0"
+solana-clap-utils = "=1.7.12"
+solana-cli-config = "=1.7.12"
+solana-client = "=1.7.12"
+solana-logger = "=1.7.12"
+solana-sdk = "=1.7.12"
+solana-program = "=1.7.12"
 spl-token-lending = { path="../program", features = [ "no-entrypoint" ] }
-spl-token = { path="../../token/program", features = [ "no-entrypoint" ]  }
+spl-token = "=3.2.0"
```

Comment:
?

Reply:
deleted the token program from the repo
pinned spl-token to v3.2.0

Reply:
y not 3.4 i mean and why remove feature